packages/vc-verification/src/issuer-def-resolver.ts

```diff
@@ -48,9 +50,18 @@ export class EthersProviderIssuerDefinitionResolver
       this._roleDefResolver.issuers(resolvedNamespace),
       this._roleDefResolver.issuerType(resolvedNamespace),
     ]);
+    const chainID = (await this._provider.getNetwork()).chainId;
+    let issuerDIDs: string[] = result.dids;
+    if (chainID == 73799) {
+      issuerDIDs = issuerDIDs.map(
+        (issuerDIDs) => `did:ethr:volta:${issuerDIDs}`
+      );
+    } else {
+      issuerDIDs = issuerDIDs.map((issuerDIDs) => `did:ethr:ewc:${issuerDIDs}`);
+    }
```

Comment:
- I would check explicitly for EWC otherwise there could be confusing results. I think it would be confusing (and technically incorrect) if a non-EWC provider is configured but EWC DIDs are returned.
- Better to import the CHAIN_IDs https://github.com/energywebfoundation/ew-credentials/blob/bug/did_type_issuer_is_not_authorized/packages/credential-governance/src/chain-constants.ts
```suggestion
    if (chainID == VOLTA_CHAIN_ID) {
      issuerDIDs = issuerDIDs.map(
        (issuerDIDs) => `did:ethr:volta:${issuerDIDs}`
      );
    } else if (chainID === EWC_CHAIN_ID) {
      issuerDIDs = issuerDIDs.map((issuerDIDs) => `did:ethr:ewc:${issuerDIDs}`);
    }
    else {
      throw Exception()
    }
```